app/services/foreman_ansible/renderer_methods.rb

```diff
@@ -9,13 +9,21 @@ def insights_remediation(plan_id, organization_id = Organization.current.id)
         plan_id
       )
       rules = insights_plan.playbook
+      disclaimer = insights_plan.parse_disclaimer
       hostname_rules_relation = insights_plan.hostname_rules(rules)
       global_rules = insights_plan.rules_to_hash(rules)
-      host_playbooks = hostname_rules_relation[@host.name].
-                       reduce([]) do |acc, cur|
+      host_playbooks = individual_host_playbooks(hostname_rules_relation,
+                                                 global_rules)
+      "#{disclaimer}\n#{host_playbooks.to_yaml}"
+    end
+
+    private
+
+    def individual_host_playbooks(hostname_rules_relation, global_rules)
+      hostname_rules_relation[@host.name].
```

Comment:
rubocop makes this really hard to read, 80 chars limitation is really (not introduced by this commit, just randomly ranting)

Reply:
That was just me, not enforced by rubocop. I'll update it & add tests

Reply:
oh, ok :-)